### ISSUE TITLE
TypeError: 'caller' and 'arguments' are restricted function properties and cannot be accessed in this context.

### ISSUE THREAD

**Issue by choopage** [postmanlabs/newman#292]
I tried using my collections and your example collection (from github.com/postmanlabs/newman); but they produced the same error message.
```
newman -c dumpcollection.json

newman -u https://www.getpostman.com/collections/4dc9f7ecb2645647b525

newman -u https://www.getpostman.com/collections/cb208e7e64056f5294e5
```

![capture](https://cloud.githubusercontent.com/assets/6467136/11230287/91bc4e3e-8dd8-11e5-8fd7-22557842343e.PNG)

I'm using newman version 1.2.23.

**Comment by czardoz** [comment]
Can you tell me your Newman version and the Node version you're using? If you're on Node v4+ you'll need to install Newman Beta.

`npm install -g newman@beta `

**Comment by choopage** [comment]
@czardoz 
I'm using newman version 1.2.23 & node version 5.
I'll switch to newman beta. 

If this instruction isn't in your README.md of newman, could you please include it? Thanks.

**Comment by czardoz** [comment]
It's there in the README: https://github.com/postmanlabs/newman/blob/develop/README.md#newman-beta

**Comment by choopage** [comment]
@czardoz 
Oh I see understand it now. You are referring to this sentence: *We've released a beta of Newman v2.0, which supports Node v4.0+ as well. If you wish to install the beta,*

I read that but couldn't make much out of it. I didn't interpret that sentence well. It's because *if you wish to install the beta* illustrates that it's an option to use newman beta.
 
My suggestion is to re-word it. *For Node version 4 & above, please use Newman beta version.*

```
npm install -g newman@beta
```

All in all, thanks for pointing me to it the right newman version. Cheers.
